### PR TITLE
I made changes to the wrong branch

### PR DESCRIPTION
**Pull request by NRKirby**
How to apply uncommitted changes to another branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -427,6 +428,18 @@ When you want to get rid of all of your untracked files
 $ git clean -f
 ```
 
+### I made changes on the wrong branch
+
+You've made uncommitted changes and realise you're on the wrong branch
```
Comment (RichardLitt) [inline README.md]: Add a period at the end of the line.

File: README.md [hunk 2]
```diff
@@ -427,6 +428,18 @@ When you want to get rid of all of your untracked files
 $ git clean -f
 ```
 
+### I made changes on the wrong branch
+
+You've made uncommitted changes and realise you're on the wrong branch 
+
+Stash changes and apply them to the branch you want
```
Comment (RichardLitt) [inline README.md]: Add a period, and make this part of the previous paragraph.

File: README.md [hunk 3]
```diff
@@ -427,6 +428,18 @@ When you want to get rid of all of your untracked files
...
+
+```
```
Comment (RichardLitt) [inline README.md]: Should be a `sh` code block.

File: README.md [hunk 4]
```diff
@@ -427,6 +428,18 @@ When you want to get rid of all of your untracked files
...
+git stash
```
Comment (RichardLitt) [inline README.md]: Please use the `(branch) $ git stash` style used elsewhere.

**Comment by NRKirby** [comment]
I've made the changes you've asked for, let me know if there's anything else @RichardLitt

File: README.md [hunk 5]
```diff
@@ -427,6 +428,16 @@ When you want to get rid of all of your untracked files
 $ git clean -f
 ```
 
+### I made changes on the wrong branch
+
+You've made uncommitted changes and realise you're on the wrong branch. Stash changes and apply them to the branch you want:
+
+```sh
+(wrong_branch)$ git stash
+(wrong_branch)$ git checkout <branch you want to be on>
```
Comment (RichardLitt) [inline README.md]: Maybe change `<branch you want to be on>` to `correct_branch`?

**Comment by RichardLitt** [comment]
Looks better! Move the section down, too - I think you changed the entry in the TOC, but didn't move the section itself.

File: README.md (hunk 5 above)
Comment (NRKirby) [reply README.md]: Done

**Comment by RichardLitt** [comment]
Looks good to me! Thank you, @NRKirby!

**Pull request merged**